config/datatypes_conf.xml.sample

```diff
@@ -187,6 +187,7 @@
     <datatype extension="searchgui_archive" type="galaxy.datatypes.binary:SearchGuiArchive" display_in_upload="True"/>
     <datatype extension="peptideshaker_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>
     <!-- End Proteomics Datatypes -->
+    <datatype extension="netcdf" type="galaxy.datatypes.binary:NetCDF" mimetype="application/octet-stream" display_in_upload="true" description="Format used by netCDF software library for writing and reading chromatography-MS data files." />
```

Comment:
Because it is sniffable you want to add a line similar to this at the bottom: https://github.com/lecorguille/galaxy/blob/ad8483a7db24a9067a2f6fdac4db3d409a8410a9/config/datatypes_conf.xml.sample#L575